.travis.yml

```diff
@@ -235,6 +235,7 @@ install:
 
 script:
   - cd ${TRAVIS_BUILD_DIR}
+  - if [ "$TRAVIS_RUST_VERSION" != "1.25.0" ]; then cargo fmt -- --check; fi
```

Comment:
I think it's better to create a separate job for this, same as "CHECKS" job for i18nspector. This way:
1. wrong formatting doesn't stop Travis from compiling the source;
2. it's easy to see what's wrong, exactly, just by looking at the list of jobs. With current setup, wrong formatting will fail almost all jobs, and it would look like everything's broken. One failed job named "formatting" will be much more informative.

Reply:
Makes perfect sense!

I changed the `CHECKS` variable to contain which check to execute so I don't have to check multiple variables for emptiness above.
I hope this is fine.

Reply:
Yup, that sounds good; thank you!

Let's fix the build, and wrap this up.